data/api/build.gradle

```diff
@@ -19,7 +19,13 @@ tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile.class).all {
 
 kotlin {
     android()
-    ios()
+
+    // Add after resolving Firebase auth dependencies
+    // if (System.getenv("SDK_NAME")?.startsWith("iphoneos")) {
+    //     iosArm64("ios")
+    // } else {
+    //     iosX64("ios")
+    // }
```

Comment:
I'm having a hard time linking Firebase auth 🤔 
Therefore, it has not been tested with the API module🙇‍♂️
In this PR, I will stop adding the iOS target to the API module, and would like to support it in another PR🙇‍♂️

Reply:
I also had a hard time linking this library before. 👍